### ISSUE TITLE
Open-CAS v21.6.4 build errors 

### ISSUE THREAD

**Issue by lxtakc** [Open-CAS/open-cas-linux#1103]
## Description

Open-CAS v21.6.4 build errors
## Expected Behavior

Successfully build and install Open-CAS.
## Actual Behavior

When I try to make Open-CAS it fails with errors. 

## Steps to Reproduce

1. Try to make Open-CAS v21.6.4

## Logs

```
make[3]: Entering directory `/home/user/open-cas-linux-21.06.4.0554.release/ocf'
make[3]: Nothing to be done for `distcleandir'.
make[3]: Leaving directory `/home/user/open-cas-linux-21.06.4.0554.release/ocf'
make[2]: Leaving directory `/home/user/open-cas-linux-21.06.4.0554.release/ocf'
cd "/lib/modules/"3.10.0-1160.59.1.el7.x86_64"/build" && make M=/home/user/open-cas-linux-21.06.4.0554.release/modules modules
make[2]: Entering directory `/usr/src/kernels/3.10.0-1160.59.1.el7.x86_64'
  CC [M]  /home/user/open-cas-linux-21.06.4.0554.release/modules/cas_cache/layer_upgrade.o
/home/user/open-cas-linux-21.06.4.0554.release/modules/cas_cache/layer_upgrade.c: In function ‘_cas_upgrade_restore_conf_main’:
/home/user/open-cas-linux-21.06.4.0554.release/modules/cas_cache/layer_upgrade.c:949:5: error: ‘struct ocf_mngt_cache_config’ has no member named ‘metadata_layout’
  cfg.metadata_layout = metadata_layout;
     ^
/home/user/open-cas-linux-21.06.4.0554.release/modules/cas_cache/layer_upgrade.c:961:12: error: ‘struct ocf_mngt_cache_device_config’ has no member named ‘cache_line_size’
  device_cfg.cache_line_size = cache_line_size;
            ^
/home/user/open-cas-linux-21.06.4.0554.release/modules/cas_cache/layer_upgrade.c:963:12: error: ‘struct ocf_mngt_cache_device_config’ has no member named ‘force’
  device_cfg.force = true;
            ^
make[4]: *** [/home/user/open-cas-linux-21.06.4.0554.release/modules/cas_cache/layer_upgrade.o] Error 1
make[3]: *** [/home/user/open-cas-linux-21.06.4.0554.release/modules/cas_cache] Error 2
make[2]: *** [_module_/home/user/open-cas-linux-21.06.4.0554.release/modules] Error 2
make[2]: Leaving directory `/usr/src/kernels/3.10.0-1160.59.1.el7.x86_64'
make[1]: *** [default] Error 2
make[1]: Leaving directory `/home/user/open-cas-linux-21.06.4.0554.release/modules'
make: *** [modules] Error 2

```

## Your Environment

* OpenCAS version (commit hash or tag): v21.6.4
* Operating System: CentOS 7
* Kernel version: 3.10.0-1160.59.1.el7.x86_64

P.S. I can make and install v21.6.1 on the same server.

**Comment by mmichal10** [comment]
Hi @lxtakc 

thank you for posting the issue. We are investigating it. 

Also, I want to let you know that a few days ago we released CAS v21.6.5. [You can download it here](https://github.com/Open-CAS/open-cas-linux/releases/tag/v21.6.5)

**Comment by robertbaldyga** [comment]
Hi @lxtakc 

Thanks for reporting the issue. We fixed v21.6.4 packages.
However, as @mmichal10 suggested, we already have v21.6.5, so I'd strongly recommend using the latest version.

**Comment by lxtakc** [comment]
Hello @robertbaldyga , @mmichal10! 

Thank you for your reply.

I have downloaded v21.6.5. I have successfully built it and can work with it.
